application/src/main/java/org/togetherjava/tjbot/commands/UserInteractor.java

```diff
@@ -18,9 +20,8 @@ public interface UserInteractor extends Feature {
     /**
      * Gets the name of the interactor.
      * <p>
-     * Requirements for this are documented in
-     * {@link net.dv8tion.jda.api.interactions.commands.build.Commands#slash(String, String)}.
-     * <p>
+     * It is not advised to start the name with {@code s-}, {@code mc-} or {@code uc-}
```

Comment:
fragile. easily gets out of date. link against some proper extracted source instead. for example an enum or similar.
also, as discussed earlier, id go for fail-fast and not even allow it

Reply:
yeah, sounds appropriate